algos/level2/week17/robert.index.js

```diff
@@ -0,0 +1,27 @@
+
+class Stack {
+    constructor(){
+        this.values = []
+    }
+}
+
+class Queue {
+    constructor() {
+       const stackA = new Stack
+       const stackB = new Stack
+     this.values = stackA.values.concat(stackB.values)
```

Comment:
Hi Robert, stackA.values and stackB.values are undefined. That's why it won't work. Try to solve it without making an array from these 2 stacks. I know it will be very inefficient without an array, but we are just making an exercise :)  Try using already existing stack methods here within add(value), remove() and peek().